api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -31,18 +31,17 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Collection;
 import java.util.Date;
 import java.util.Locale;
 import java.util.Properties;
 import java.util.SortedSet;
 import java.util.TreeSet;
-import java.util.Collection;
 
 import org.apache.commons.io.IOUtils;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-
```

Comment:
@sherrif10 try to remove these empty spaces

Reply:
ok @cliff am going to do that